duckduckgo/Android
# Hotfix 5.128.1

Task/Issue URL: https://app.asana.com/0/488551667048375/1202400886433010

### Description

Update Dax dialog copy

### Steps to test this PR

- Uninstall previous app or clear data
- Install from this branch
- Make a search in DDG
- [x] Check dax cta dialog copy is "Your DuckDuckGo searches are anonymous. Always. 🙌"
